sdk/core/test/integration/NetworkMember.test.ts

```diff
@@ -1047,4 +1048,45 @@ describe('CommonNetworkMember', () => {
     // @ts-ignore
     expect(signedCredentials[0].credentialSubject.data.email).to.eq(decoded.email)
   })
+
+  it('#saveCredentials', async () => {
+    const credentials = [credential]
+    const commonNetworkMember = new CommonNetworkMember(password, encryptedSeedElem, options)
+
+    const result = await commonNetworkMember.saveCredentials(credentials)
+
+    expect(result[0].credentialId).to.eql(credential.id)
+  })
+
+  it('#getCredentialById', async () => {
+    const commonNetworkMember = new CommonNetworkMember(password, encryptedSeedElem, options)
+
+    const result = await commonNetworkMember.getCredentialById(credential.id)
+    
+    const parsedCredential = JSON.parse(result)
+    expect(parsedCredential.id).to.eql(credential.id)
+  })
+
+  it('#getAllCredentials', async () => {
+    const commonNetworkMember = new CommonNetworkMember(password, encryptedSeedElem, options)
+
+    const result = await commonNetworkMember.getAllCredentials([])
+
+    const parsedCredential = JSON.parse(result[0])
+    expect(parsedCredential.id).to.eql(credential.id)
+  })
+
+  it('#getAllCredentials by types', async () => {
+    const commonNetworkMember = new CommonNetworkMember(password, encryptedSeedElem, options)
+
+    const result = await commonNetworkMember.getAllCredentials([credential.type])
+
+    const parsedCredential = JSON.parse(result[0])
+    expect(parsedCredential.id).to.eql(credential.id)
+  })
+
+  it('#deleteCredentialById', async () => {
```

Comment:
All these tests only check that `CommonNetworkMember` works with the new vault.

They do not check that `CommonNetworkMember` works with both vaults (that it returns credentials from both vaults, that it looks for a credential with requested ID in bloom vault if not found in affinidi vault; that it deletes credential from bloom vault if not found in affinidi vault)